neat_addr.h

```diff
@@ -38,6 +39,8 @@ struct neat_addr {
     uint8_t __pad;
     uint16_t __pad2;
     uint8_t prefix_length;
+
+    struct pvds pvds;
```

Comment:
I meant the opposite. That there is a struct in neat_pvd.h that stores a pointer to the address and the result of the PVD query. Not sure if it makes sense, but then again this code will be changed when the policy manager is up and running. Until the PM is ready, I would like to isolate this code as much as possible. So I suggest we have a struct pvd_res (or something) in neat_pvd.h that contains the PVD-list and a pointer to the address. If tying the PVD to an address is required, then a least make this a pointer to the pvd_res struct.